packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -21,18 +21,30 @@ enum RecoveryPhoneStatus {
   FAILURE = 'failure',
 }
 
+export type Customs = {
+  check: (req: AuthRequest, email: string, action: string) => Promise<void>;
+};
+
 class RecoveryPhoneHandler {
   private readonly recoveryPhoneService: RecoveryPhoneService;
 
   constructor(
+    private readonly customs: Customs,
     private readonly log: AuthLogger,
     private readonly glean: GleanMetricsType
   ) {
     this.recoveryPhoneService = Container.get(RecoveryPhoneService);
   }
 
   async sendCode(request: AuthRequest) {
-    const { uid } = request.auth.credentials as SessionTokenAuthCredential;
+    const { uid, email } = request.auth
+      .credentials as SessionTokenAuthCredential;
+
+    if (!email || !uid) {
+      throw AppError.invalidToken();
```

Comment:
I don't think this is possible.  Unless the function is called during a request that's not auth'd with a session token.

Reply:
I can take out the check if you think it's unnecessary.